service/population.go

```diff
@@ -0,0 +1,54 @@
+package service
+
+import (
+	"github.com/WorkWorkWork-Team/gov-ec-api/model"
+	"github.com/WorkWorkWork-Team/gov-ec-api/repository"
+	"github.com/sirupsen/logrus"
+)
+
+type populationService struct {
+	repository repository.PopulationRepository
+}
+
+type PopulationService interface {
+	GetPopulationStatistics() ([]model.PopulationResponseItem, error)
+}
+
+func NewPopulationService(populationRepository repository.PopulationRepository) PopulationService {
+	return &populationService{
+		repository: populationRepository,
+	}
+}
+
+func (p *populationService) GetPopulationStatistics() ([]model.PopulationResponseItem, error) {
+	out := []model.PopulationResponseItem{}
+	districts, err := p.repository.QueryAllDistrict()
+	if err != nil {
+		logrus.Error(err)
+	}
+	for _, s := range districts {
+		districtId := s.LocationID
+		districtName := s.Location
+		total, err := p.repository.QueryTotalPopulation(districtId)
+		if err != nil {
+			logrus.Error(err)
+		}
+		haveRight, err := p.repository.QueryPeopleRightToVote(districtId)
+		if err != nil {
+			logrus.Error(err)
+		}
+		commit, err := p.repository.QueryPeopleCommitedTheVote(districtId)
+		if err != nil {
+			logrus.Error(err)
+		}
```

Comment:
Should we skip to insert this result in array if some error happen during these call?

Reply:
What about this?

Reply:
Fixed